assets/src/components/Sidebar/ThemeSettings.js

```diff
@@ -55,7 +55,7 @@ export const ThemeSettings = props => {
   const { editPost } = useDispatch('core/editor');
 
   useEffect(() => {
-    handleThemeSwitch('theme', meta.theme, meta)
+    handleThemeSwitch('theme', meta.theme, meta);
```

Comment:
Same comment here as earlier, not a blocker but we can probably shorten this as follows:
```
useEffect(() => handleThemeSwitch('theme', meta.theme, meta), [meta.theme]);
```

Reply:
Fixed